elastic/beats
# Nats module improvements

## What this PR adds
This PR adds more metrics at `connections` and `routes` metricsets. Prior to this no metrics have been collected for each of the connections and for each of the routes. With this PR we collect metrics like in/out bytes etc for each the connections and for each of the routes. In order to have routes in a NATS server a more than 1 NATS servers should be presented creating a NATS cluster. For this docker-compose file of the module adds an extra NATS instance so as populate routes and their respective metrics. 

## Why it is important
It related to https://github.com/elastic/integrations/issues/359. Next steps would be to enhance/improve dashboards.

## How to test this manually

1. Start the nats cluster using `docker-compose` file of nats module
2. Enable nats metricbeat module `connections` and `routes` metricsets and point them to the monitoring port of the `nats` container (exported port of 8222 internal)
3. Make sure that connection and route details are being collected.
